User-Stories/PO-PM-User-Stories/PO-user-stories.md

```diff
@@ -0,0 +1,60 @@
+#Product Owner (PO) / Project Manager (PM) user stories
```

Comment:
As already said I do not know whether product owner is also a term used in organizations other than companies

Reply:
What would, from your opinion, be an alternative and widely spread / understood role name suitable for this role

Reply:
project manager could be an alternative

Reply:
OK. will change to project manager. Guess it will not make much of difference. In the mapping I will keep the Product Owner for those, who have this role in place.

Reply:
Fine